crypto/fipsmodule/service_indicator/service_indicator.c

```diff
@@ -217,6 +217,39 @@ static int is_md_fips_approved_for_verifying(int md_type, int pkey_type) {
   }
 }
 
+// custom_meth_invoked checks whether custom crypto was invoked in the |meth|
+// or |eckey_method| fields for a given |RSA| or |EC_KEY| respectively. For
+// |RSA| keys, custom verify and sign functionality is supported. For |EC_KEY|
+// keys, only custom sign functionality is supported.
+// Returns one if no custom crypto was invoked and zero otherwise.
+static int no_custom_meth_invoked(const EVP_PKEY_CTX *ctx) {
+  const int pkey_type = EVP_PKEY_id(ctx->pkey);
+  if(pkey_type == EVP_PKEY_RSA || pkey_type == EVP_PKEY_RSA_PSS) {
+    const RSA_METHOD *meth = ctx->pkey->pkey.rsa->meth;
+    if (meth == NULL) {
```

Comment:
`meth` is always non-NULL even if no custom methods are defined?

Reply:
yep you are correct, added the check as force of habit